python/semantic_kernel/connectors/ai/open_ai/services/open_ai_text_embedding.py

```diff
@@ -66,21 +66,22 @@ async def generate_embeddings_async(
             batch_size = batch_size or len(texts)
             for i in range(0, len(texts), batch_size):
                 batch = texts[i : i + batch_size]
-                response: Any = await openai.Embedding.acreate(
-                    **model_args,
+                client = AsyncOpenAI(
                     api_key=self._api_key,
-                    api_type=self._api_type,
-                    api_base=self._endpoint,
-                    api_version=self._api_version,
+                    base_url=self._endpoint,
                     organization=self._org_id,
+                    version=self._api_version
+                )
+                response: Any = await client.embeddings.create(
+                    **model_args,
                     input=batch,
                 )
                 # make numpy arrays from the response
-                raw_embeddings.extend([array(x["embedding"]) for x in response["data"]])
+                raw_embeddings.extend([array(x.embedding) for x in response.data])
             return array(raw_embeddings)
         except Exception as ex:
             raise AIException(
                 AIException.ErrorCodes.ServiceError,
-                f"{self.__class__.__name__} failed to generate embeddings",
+                "OpenAI service failed to generate embeddings",
```

Comment:
I think `f"{self.__class__.__name__}` was added previously because when developers used `AzureTextEmbedding` class which inherits `OpenAITextEmbedding` class, in case of exception, the error message will say `OpenAI service failed to...`, but it was actually `Azure OpenAI` service, so it was confusing.

Issue: https://github.com/microsoft/semantic-kernel/issues/3171
PR: https://github.com/microsoft/semantic-kernel/pull/3172